datashuttle/command_line_interface.py

```diff
@@ -235,7 +192,7 @@ def update_config(project, args):
     option_key = kwargs["option_key"]
```

Comment:
the option_key is currently required to be in the python conventional syntax, i.e. with underscor (e.g. remote_path).
Is there any way to also make it accept the alias (e.g. remote-path) when it is being callsed from the command line (given that in almost all other parts of the CLI both options work)?

Reply:
nice idea thanks!